cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -109,7 +109,7 @@
 
   INSTANCE_URI("cdap-instance-uri"),
   VERIFY_SSL_CERT("verify-ssl-cert"),
-  ENTITY("entity-id"),
```

Comment:
I think that it is a placeholder for a value, rather than a value or parameter itself. I will test the change.